schemaloaders/schemaloaders.go

```diff
@@ -67,36 +65,19 @@ func (s *SchemaLoader) Load() error {
 		ms = append(ms, featurecompletions.FromFile(nil, &s.schema.FeatureCompletions[i]))
 	}
 
-	newSchema := &serializers.Schema{
-		SerializerVersion: serializers.SerializerVersion,
-		SchemaVersion:     s.schema.SchemaVersion,
-	}
 	for _, migration := range ms {
-		err := migrationmanagers.NewWithDependencies(migration, s.server, newSchema).Apply(migrations.Repository{}) // generated migrations are data-complete and don't need migrations
-		if err != nil {
-			return err
-		}
-	}
-
-	schema.SortAlphabetically(newSchema)
-	if !reflect.DeepEqual(*s.schema, *newSchema) {
-		before, err := yaml.Marshal(s.schema)
-		if err != nil {
-			return err
-		}
-		after, err := yaml.Marshal(newSchema)
+		err := migrationmanagers.NewWithServer(migration, s.server).Sync()
```

Comment:
This diff is hard to read, the individual last two commits may make it easier to see what's going on. Thanks golang biolerplatey-error-handling for making github detect similarities between different codepaths.